app/views/eduHeader.html

```diff
@@ -1,35 +1,23 @@
-<div>
-  <div id="main-header">
-    <div translate="dashboard.title">Education Dashboard</div>
-  </div>
-  <lang-selector></lang-selector>
-
-  <nav class="navbar navbar-default" id="main-nav">
-    <div class="container-fluid">
-      <div class="navbar-header">
-        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#main-nav-items">
-          <span class="sr-only">Toggle navigation</span>
-          <span class="icon-bar"></span>
-          <span class="icon-bar"></span>
-          <span class="icon-bar"></span>
-        </button>
-      </div>
+<div id="main-header">
+  <div class="container-fluid">
+    <div class="navbar-brand header-flag"><img src="images/tzflag.png"></div>
```

Comment:
make sure to always include `alt=` attribute on `<img />` tags. @esoliani how do we translate HTML tag attributes?